diff --git a/.env.development b/.env.development
index 1be54ee..275f14d 100644
--- a/.env.development
+++ b/.env.development
@@ -1,2 +1,2 @@
 NEXT_PUBLIC_ENV=development
-NEXT_PUBLIC_API=https://api-testnet-public.algodex.com
+NEXT_PUBLIC_API=https://api-testnet.algodex.com
diff --git a/.env.production b/.env.production
index a81b909..ffd6cd3 100644
--- a/.env.production
+++ b/.env.production
@@ -1,2 +1,2 @@
 NEXT_PUBLIC_ENV=production
-NEXT_PUBLIC_API=https://api-testnet-public.algodex.com
+NEXT_PUBLIC_API=https://api-testnet.algodex.com
diff --git a/lib/api.js b/lib/api.js
index 55945f4..d5e60ec 100644
--- a/lib/api.js
+++ b/lib/api.js
@@ -38,7 +38,7 @@ export async function fetchTradeHistoryByAddress(walletAddress, includeAssetInfo
 
 export async function fetchAssetsByByAddress(walletAddress) {
   const res = await axios.get(
-    `${API_HOST}/algodex-backend/wallet_assets.php?ownerAddr=${walletAddress}`
+    `https://api-testnet.algodex.com/algodex-backend/wallet_assets.php?ownerAddr=${walletAddress}`
   )
   return res.data
 }
diff --git a/services/order.js b/services/order.js
index f3c6f6d..1a75596 100644
--- a/services/order.js
+++ b/services/order.js
@@ -1,7 +1,7 @@
 import algodex from '@algodex/algodex-sdk'
 import { convertToBaseUnits, convertToAsaUnits } from './convert'
 
-const AlgodClient = new algodex.initAlgodClient('public_test')
+const AlgodClient = new algodex.initAlgodClient('test')
 
 const OrderService = {
   placeOrder: (order, orderBook) => {
diff --git a/services/wallet.js b/services/wallet.js
index f1da917..d472200 100644
--- a/services/wallet.js
+++ b/services/wallet.js
@@ -9,7 +9,7 @@ const WalletService = {
     }
 
     try {
-      const AlgodClient = new algodex.initAlgodClient('public_test')
+      const AlgodClient = new algodex.initAlgodClient('test')
 
       const promises = addresses.map(async (address) => {
         const accountInfo = await AlgodClient.accountInformation(address).do()
